lib/rules/max-lines-per-function.js

```diff
@@ -190,11 +193,24 @@ module.exports = {
 
             if (lineCount > maxLines) {
                 const name = upperCaseFirst(astUtils.getFunctionNameWithKind(funcNode));
+                const linesExceed = lineCount - maxLines;
+                const srcCodeLinesLength = [...sourceCode.lines].pop().length;
+
+                const loc = {
+                    start: {
+                        line: lines[maxLines].lineNumber,
+                        column: 0
+                    },
+                    end: {
+                        line: sourceCode.lines.length,
+                        column: srcCodeLinesLength
+                    }
```

Comment:
```suggestion
                    end: node.loc.end
```

Similar to the previous, `loc.end` would always be the last position in the file, while we probably want the last position in the function.